pm_yaml_config/pm_yaml_config.xml

```diff
@@ -1,138 +1,159 @@
+        <param type="text" argument="out_folder" optional="true" name="out_folder" value="muon-airss-out" label="Output folder name" help="name to call the output folder used to store the input files that the script generates."/>
+        <param type="float" argument="geom_force_tol" name="geom_force_tol" value="0.05" label="Geometry optimization tolerance" help="Tolerance on geometry optimisation in units of eV/Å."/>
+        </section>
+        <section name="calculator_params" expanded="true" title="Calculator parameters">
         <conditional name="calculator_cond">
-            <param type="select" display="radio" label="optimization calculator" name="calculator" value="uep" help="calculator to generate structure files for. Must be a single word or a comma seperated list of values. Currently supported calculators are CASTEP, DFTB+ and UEP. Can also pass all as an option to generate files for all calculators.">
-                <option value="uep">uep</option>
-                <option value="castep">castep</option>
-                <option value="dftb+">dftb</option>
+            <param type="select" display="radio" label="Optimization calculator" argument="calculator" name="calculator" value="uep" help="Calculator to generate structure files for. Must be a single word or a comma seperated list of values. Currently supported calculators are CASTEP, DFTB+ and UEP. Can also pass all as an option to generate files for all calculators.">
+                <option value="uep">UEP</option>
+                <option value="castep">CASTEP</option>
+                <option value="dftb+">DFTB+</option>
                 <option value="all">all</option>
             </param>
             <when value="uep">
-                <param type="float" name="uep_gw_factor" optional="true" value="4.0" label="gaussian witdh factor" help="Gaussian width factor for UEP calculation. Higher values will make the potential of atomic nuclei closer to the point-like approximation but may introduce artifacts."/>
-                <param type="text" name="uep_chden" optional="true" label="path to .den_fmt and castep files" help="Name of the .den_fmt file containing the electronic density for an Unperturbed Electrostatic Potential optimisation. The corresponding .castep file must be with the same seedname."/>
+                <param type="float" argument="uep_gw_factor" name="uep_gw_factor" optional="true" value="4.0" label="Gaussian witdh factor" help="Gaussian width factor for UEP calculation. Higher values will make the potential of atomic nuclei closer to the point-like approximation but may introduce artifacts."/>
+                <param type="text" argument="uep_chden" name="uep_chden" optional="true" label="Path to .den_fmt and CASTEP files" help="Name of the .den_fmt file containing the electronic density for an Unperturbed Electrostatic Potential optimisation. The corresponding .castep file must be with the same seedname. Only required for use outside Muon Galaxy"/>
```

Comment:
```suggestion
                <param type="text" argument="uep_chden" name="uep_chden" optional="true" label="Path to .den_fmt and .castep files" help="Name of the .den_fmt file containing the electronic density for an Unperturbed Electrostatic Potential optimisation. The corresponding .castep file must be with the same seedname. Only required for use outside Muon Galaxy"/>
```

(Let's leave discussions about whether this is too advanced an argument for another issue.)